uavcan/equipment/air_data/1027.RawAirData.uavcan

```diff
@@ -0,0 +1,41 @@
+#
+# Raw Air Data.
+#
+
+# Note: unused vars should be assigned NaN
+
+#
+# Heater State
+# 
+uint8 FLAG_HEATER_AVAILABLE      = 1
+uint8 FLAG_HEATER_WORKING        = 2
+uint8 FLAG_HEATER_OVERCURRENT    = 4
+uint8 FLAG_HEATER_OPENCIRCUIT    = 8
+uint8 flags
+
+#
+# Pressure Data
+#
+float32 static_pressure                 # Pascal
+float32 differential_pressure           # Pascal
+
+#
+# Temperature Data
+#
+float16 static_p_sensor_temperature     # Kelvin
+float16 diff_p_sensor_temperature       # Kelvin
```

Comment:
@pavel-kirienko Just feel that its too long var name and I don't see any additional info being given by increasing it to that length.